### PR TITLE
Add installation guide for Feast on Azure Kubernetes Service (AKS)

### PR DESCRIPTION
**Pull request by xiaoyongzhu**
Installation guide for Feast on AKS. Have completed the whole process by working with @jklegar .

**What this PR does / why we need it**:

Provide installation guide for Feast on Azure Kubernetes Service (AKS)

**Which issue(s) this PR fixes**:

Fixes #

None

**Does this PR introduce a user-facing change?**:

No
```release-note
Add installation guide for Feast on Azure Kubernetes Service (AKS)
```

### CODE REVIEW

File: docs/getting-started/install-feast/kubernetes-azure-aks-with-helm.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+### Overview
+
+This guide installs Feast on Azure Kubernetes cluster (known as AKS), and ensures the following services are running:
+
+* Feast Core
+* Feast Online Serving
+* Postgres
+* Redis
+* Spark
```
Comment (jklegar) [inline docs/getting-started/install-feast/kubernetes-azure-aks-with-helm.md:11]: also Kafka

**Comment by jklegar** [comment]
Hey @xiaoyongzhu, thanks for contributing! A couple small comments:
- what do you think about combining this page with the Kubernetes with Helm guide? I'm wondering if that will be easier for users to understand and make the docs lower maintenance. I'll leave it to your call, but wanted to raise the question
- if you go with a separate guide, can you add it to docs/SUMMARY.md?

thanks

**Review by woop — APPROVED** [review]

File: docs/getting-started/install-feast/kubernetes-azure-aks-with-helm.md (hunk 1 above)
Comment (xiaoyongzhu) [reply docs/getting-started/install-feast/kubernetes-azure-aks-with-helm.md:11]: Thanks Jacob!

**Comment by xiaoyongzhu** [comment]
> Hey @xiaoyongzhu, thanks for contributing! A couple small comments:
> 
> * what do you think about combining this page with the Kubernetes with Helm guide? I'm wondering if that will be easier for users to understand and make the docs lower maintenance. I'll leave it to your call, but wanted to raise the question
> * if you go with a separate guide, can you add it to docs/SUMMARY.md?
> 
> thanks

Thanks! I looked thru the doc again and feel there might be some Azure specific configs (like BLOB key etc.) so I might suggest we keep it a separate doc. I've updated the SUMMARY.md file per suggestion.

**Comment by xiaoyongzhu** [comment]
@woop I feel it's ready to be merged (not sure if all those unit tests will be passed though). LMK if you have anything else from me!

**Comment by woop** [comment]
> @woop I feel it's ready to be merged (not sure if all those unit tests will be passed though). LMK if you have anything else from me!

Thanks @xiaoyongzhu

**Review by woop — APPROVED** [review]

**Pull request merged**